modules/runner/README.md

```diff
@@ -2,7 +2,7 @@
 This document describes user-level and component-level cluster lifecycles and their mutual interaction.
 
 ## Node lifecycle
-A node maintains its' local state in the local persistent key-value storage named vault. The data stored in the vault is 
+A node maintains its local state in the local persistent key-value storage named vault. The data stored in the vault is 
```

Comment:
Could you also please remove the part about the metastorage projected properties?

Reply:
It seems that there are more parts that need to be updated, I filed an issue: https://issues.apache.org/jira/browse/IGNITE-21294